srtcore/core.cpp

```diff
@@ -1050,6 +1050,16 @@ void CUDT::getOpt(SRT_SOCKOPT optName, void *optval, int &optlen)
         optlen             = sizeof(int64_t);
         break;
 
+    case SRTO_INPUTBW:
+       *(int64_t*)optval = m_llInputBW;
+       optlen             = sizeof(int64_t);
+       break;
+
+    case SRTO_OHEADBW:
+        *(int *)optval = m_iOverheadBW;
+        optlen = sizeof(int);
```

Comment:
doc says int32_t but int is used here. I think this is an old error not introduced in this PR.

Reply:
There was some cleaning in #1385 
The doc states:

> Possible types of socket options are:
> * `int32_t` - This type can usually be treated as an `int` equivalent since it 
does not change size on 64-bit systems. For clarity, options use this fixed size 
integer. In some cases the value is expressed using an enumeration type (see below).

I've noticed `SRTO_RETRANSMITALGO` has `int` in the table instead of `int32_t` Will fix.